apps/next/src/pages/api/.well-known/apple-app-site-association.ts

```diff
@@ -1,6 +1,7 @@
 import type { NextApiRequest, NextApiResponse } from "next";
 
-const isDevelopment = process.env.STAGE === "development";
+const isDevelopment =
+  process.env.NEXT_PUBLIC_WEBSITE_DOMAIN === "dev.showtime.xyz";
```

Comment:
sure it should not also contain ` || process.env.STAGE` as OR operator?

Reply:
Actually, `STAGE` doesn't seem to be visible. It's value was undefined. So i thought of using the NEXT_PUBLIC* variable 😅. will dig and improve it. STAGE should be visible since it's running the code server side. But i need to check what's wrong